content/docs/command-reference/repro.md

```diff
@@ -253,8 +253,8 @@ If we now run `dvc repro`, we should see this:
 ```dvc
 $ dvc repro
 Stage 'filter' didn't change, skipping
-Running stage 'count' with command:
-        python process.py numbers.txt > count.txt
+Running stage 'count':
+$ python process.py numbers.txt > count.txt
```

Comment:
This is the actual output? Including a `$` character?

Reply:
yes, it has changed from the previous "with command:  \t {command}" to this with line beginning with `$`

Reply:
So repro becomes some sort of background process that actually enters commands into the user's shell? Or we specifically print the `$` character + command string?

Reply:
we print it. Indeed it went from previous repeated "running command", to "> {command}" suggested by @skshetry to "$ {command}" because the ">" were confusing with possible ">" inside a command (like echo > file)

Reply:
@ClementWalter @skshetry @efiop this can create a mess across all docs where we use `$` as a starting symbol for all shell commands. We'll need some mechanism to escape it, and even if we do we'll end up with confusing text anyway - mix of actual commands and outputs. Let's reconsider this please. Also out of curiosity how exactly is it related to the PR (multiple commands), I mean this particular change in UI/UX? what was the reason to change it?

Reply:
Also, it's better to make it in a way that you can copy the whole line (command) by clicking on it. In docs engine made in way to ignore $, in case or suggested output in repro $ will go into command - it might be annoying.

Reply:
@ClementWalter, could you please rollback `$ command` to be something like the following:
```console
Running stage 'build':
  echo hello
  echo world
```

We can discuss a better UI later (`>` vs indent, etc.), but the `$` sign seems to be not possible in the docs and will make users confuse with the shell. 

Also could just do either of https://github.com/iterative/dvc/pull/4934#discussion_r533048696 as it's a 2.0 change. WDYT @shcheklein?

Reply:
@skshetry I'm totally fine with `>` for now. Don't see any other good options to be honest how to keep it "copyable" and at them moment look different then output. 

A few possible options can be: 

- use delimiters (like testing systems do). Can be noisy
- use colors? Highlight commands somehow?

Reply:
Maybe similar as before, saying "with commands:" and using tabs? Other commands use tabs (e.g. `status` so that would be consistent). Up to you, just lmk please. so I can finish up the docs.

Reply:
p.s. not sure `>` is great since it can be part of the command, as above. Updated this with a tab for now.

Reply:
Implemented this on https://github.com/iterative/dvc/pull/5041.

> not sure > is great since it can be part of the command

For now, we add `>` instead of `$`. Let's discuss this on a different issue.

Reply:
> Let's discuss this on a different issue.

Created https://github.com/iterative/dvc/issues/5043 to discuss this further.

Reply:
> For now, we add > instead of $.

Actually right now it's a tab, why change it if we're not sure? I'd vote to roll back that change and yes to the discussion.

Reply:
Nvmd. Per https://github.com/iterative/dvc/discussions/5043#discussioncomment-198313 I have no strong opinion on this.

Reply:
Actually I'm back to having an opinion 😆 — see https://github.com/iterative/dvc/discussions/5043#discussioncomment-198360